geowebcache-webapp/src/test/java/org/georchestra/geowebcache/GeorchestraURLManglerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.georchestra.geowebcache;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class GeorchestraURLManglerTest {
+
+    @Test
+    public void testGeorchestraURLMangler() {
```

Comment:
I am the original author of these tests but I am wondering by reading of them once again if they don't expose some flaws: ";" and multiple slashes in the URL might now be rejected by spring security filters.

Reply:
woot... dunno, open a separate issue?